src/scheduler.rs

```diff
@@ -24,6 +26,7 @@ pub fn run_scheduler(
     redis_host: String,
     config_loaded_receiver: Receiver<BootResult>,
     region: String,
+    redis_enable_cluster: bool,
```

Comment:
feels like there might be some smell passing this around, but I think it's fine for now.

Reply:
Yeah, feels like maybe we should pass a redis config object around or something?

Reply:
Yeah or some preconfigured object that can instantiate clients that knows about the config